sympy/core/basic.py

```diff
@@ -329,7 +329,7 @@ def __ne__(self, other):
 
            but faster
         """
-        return not self.__eq__(other)
+        return not self == other
```

Comment:
btw, this method redundant, isn't?

from py3 docs:
"By default, __ne__() delegates to __eq__() and inverts the result unless it is NotImplemented."